nbs/tsdataset.ipynb

```diff
@@ -54,11 +54,12 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #83.            ids = np.array([])
</pre><p>Same for this, we should have a list, append to it and call pd.Index on it at the end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Sounds good, I've just kept it as a pd.Series though to make the type consistent with the indices in the from_df() method in the TimeSeriesDataset